From 33127ffd94abff77ba370c53548ab5f32e625637 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Thu, 18 Dec 2014 11:59:30 -0500
Subject: [redhat] kabi: Adding symbol pnp_register_driver (drivers/pnp/driver.c)

Message-id: <67361f6a4b0781cee348af602bd3090c252c600a.1418903403.git.poros@redhat.com>
Patchwork-id: 102268
O-Subject: [PATCH RHEL7.1 38/48] kabi: Adding symbol pnp_register_driver (drivers/pnp/driver.c)
Bugzilla: 1160754
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1160754
    https://bugzilla.redhat.com/show_bug.cgi?id=1160754

    Description:
    ============
    The large chuncks of memory that are reserved to be superpages
    are defined as reserved to the BIOS and passed to the the superpages
    module as an ACPI device, "superpages". This symbol is on of several
    required to handle that device.
    Used in superpages.ko

Symbol History
==============
 Request:
 ========
 [PATCH RHEL7 BZ1062669 47/49] kabi: Adding symbol pnp_register_driver (drivers/pnp/driver.c)
 Jiri Olsa <jolsa@redhat.com> (Fri, 14 Feb 2014 14:24:09 +0100)
 Description:
 ============
 The large chuncks of memory that are reserved to be superpages
 are defined as reserved to the BIOS and passed to the the superpages
 module as an ACPI device, "superpages".   These symbols are
 required to handle that device.

 Reviews:
 ========
 N/A

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/pnp_register_driver

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_x86_64/pnp_register_driver b/redhat/kabi/kabi-module/kabi_x86_64/pnp_register_driver
new file mode 100644
index 0000000..22731d8
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/pnp_register_driver
@@ -0,0 +1,2 @@
+#1-
+0xc0bf66f6 pnp_register_driver vmlinux EXPORT_SYMBOL
-- 
1.7.1